src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        for (let j = 0; j < action.keysToRemove.length; j++) {
+          delete state[action.keysToRemove[j]];
+        }
```

Comment:
please use `for of` loop or `forEach` in that case

Reply:
using normal for loop with index is something bad  ?

Reply:
@Sancz0pansa it's not bad, but it's more readable to use `for of` if you want to get each value anyway. 

e.g.
```js 
const example = [1, 2, 3];

for (let i =0; i < example.length; i++) {
  const value = example[i]
}
```

vs


```js 
const example = [1, 2, 3];

for (const value of example) {
  // I have a value and the code is readable
}
```